main.go

```diff
@@ -74,7 +77,7 @@ func extractEventsAsLinks(events GithubEvents, d3Data *D3) {
 
 // buildGraph wraps around the other graph-building functions to generate new graph data.
 // It iterates on as many API event pages as specified via the CLI flag or the default value.
-func buildGraph() bool {
+func buildGraph(nextRefresh int64) (string, bool) {
```

Comment:
+ Error handling has been updated, no more booleans returned - the signature can now be simplified.
+ Dashboard data should not be included in graph-related methods. A separate JSON file should be served in another dedicated route in `http.go`.